libnavigation-base/src/test/java/com/mapbox/navigation/base/route/NavigationRouteTest.kt

```diff
@@ -71,7 +73,7 @@ class NavigationRouteTest {
             FileUtils.loadJsonFixture("multileg_route.json")
         )
 
-        val navigationRoute = directionsRoute.toNavigationRoute()
+        val navigationRoute = directionsRoute.toTestNavigationRoute(RouterOrigin.Custom())
```

Comment:
Maybe use `RouterOrigin.Custom()` as a default parameter?
```suggestion
        val navigationRoute = directionsRoute.toTestNavigationRoute()
```